library/.pre-commit-config.yaml

```
@@ -0,0 +1,17 @@
+repos:
+  - repo: https://github.com/Lucas-C/pre-commit-hooks
+    rev: v1.4.2
+    hooks:
+    -   id: insert-license
+        name: "Insert license header in Python source files"
+        files: \.py$
+        args:
+            - --license-filepath
```

yamllint (1.35.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)